doc/modules/cassandra/pages/troubleshooting/index.adoc

```diff
@@ -8,10 +8,7 @@ isolate the problem using logs and tools. Luckily Cassandra had a great
 set of instrospection tools to help you.
 
 These pages include a number of command examples demonstrating various
-debugging and analysis techniques, mostly for Linux/Unix systems. If you
-don't have access to the machines running Cassandra, or are running on
```

Comment:
Would keep the bit about "If you don't have access to the machines running C*" etc etc